src/components/FormattedDictionary.js

```diff
@@ -0,0 +1,19 @@
+import { withNamespaces } from 'react-i18next';
+
+const looksLikeTranslation = value => typeof value === 'string';
+
+const FormattedDictionary = ({ t, options = {}, children, ...ids }) => {
+  const translations = {};
+
+  Object.entries(ids).forEach(([key, value]) => {
+    if (looksLikeTranslation(value))
+      translations[key] = t(value, options[key] || {});
```

Comment:
For these components we've been separating out the formatting logic from that view layer logic. Components that connect formatting to the React live here: https://github.com/ezcater/react-i18next-components/tree/master/src/formatters and the actual business logic lives in formatting functions here: https://github.com/ezcater/react-i18next-components/tree/master/src/formatters. Would it be worth creating a `formatDictionary` function or re-using the `formatMessage` function?